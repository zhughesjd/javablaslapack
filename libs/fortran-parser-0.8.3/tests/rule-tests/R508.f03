! Testing access-spec, R508
TYPE, PUBLIC :: A
END TYPE
type, private :: a
end type
end
