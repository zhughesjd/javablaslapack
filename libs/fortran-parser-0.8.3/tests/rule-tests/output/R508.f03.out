R1101:main-program__begin:
R 431:type-attr-spec-list__begin:
R 508:access-spec: keyword=[@5,34:39='PUBLIC',<142>,2:6] type=803
R 431:type-attr-spec: keyword=null id=null type=1200
R 431:type-attr-spec-list: count=1
R 430:derived-type-stmt: keyword=[@2,28:31='TYPE',<159>,2:0] id=[@9,44:44='A',<206>,2:16] eos=[@10,45:45='\n',<4>,2:17] hasTypeAttrSpecList=true hasGenericNameList=false
R 433:end-type-stmt: id=null
R 431:type-attr-spec-list__begin:
R 508:access-spec: keyword=[@18,61:67='private',<138>,4:6] type=804
R 431:type-attr-spec: keyword=null id=null type=1200
R 431:type-attr-spec-list: count=1
R 430:derived-type-stmt: keyword=[@15,55:58='type',<159>,4:0] id=[@22,72:72='a',<206>,4:17] eos=[@23,73:73='\n',<4>,4:18] hasTypeAttrSpecList=true hasGenericNameList=false
R 433:end-type-stmt: id=null
R 204:specification-part: numUseStmts=0 numImportStmts=0 numDeclConstructs=2
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=false hasInternalSubprogramPart=false
